interface/main/dated_reminders/dated_reminders_add.php

```diff
@@ -59,6 +59,9 @@
 // default values for $this_message    
     $this_message = array('message'=>'','message_priority'=>3,'dueDate'=>'');
     $forwarding = false;
+
+// default values for Max words to input in a reminder
```

Comment:
Why is this better than `<textarea maxlength="160" ...` which will prevent the source of error and provide immediate feedback to the user?  Then even the POST check will not be necessary since it does not look like the code is geared to accept any async messages.